coderd/workspacebuilds.go

```diff
@@ -254,6 +274,19 @@ func (api *API) workspaceBuildByBuildNumber(rw http.ResponseWriter, r *http.Requ
 	httpapi.Write(ctx, rw, http.StatusOK, apiBuild)
 }
 
+// @Summary Create workspace build
+// @ID create-workspace-build
+// @Security CoderSessionToken
+// @Accepts json
+// @Produce json
+// @Tags Builds
+// @Param id path string true "Workspace ID" format(uuid)
+// @Param request body codersdk.CreateWorkspaceBuildRequest true "Create workspace build request"
+// @Success 200 {object} codersdk.WorkspaceBuild
+// @Router /workspaces/{id}/builds [post]
+//
+// Azure supports instance identity verification:
+// https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service?tabs=linux#tabgroup_14
```

Comment:
Could we put this on top? It'd be nice to stay consistent with comment at top and swagger below. Alternatively the other way around but not ideal for Go docs on exported methods.

Reply:
Fixed.

I have also added it to the list of swagger comment validation items.